src/base/system_parameter.c

```diff
@@ -695,6 +695,7 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_PAGE_SERVER_HOSTS "page_server_hosts"
 #define PRM_NAME_SERVER_TYPE "server_type"
+#define PRM_NAME_LOG_RECOVERY_REDO_WORKER_COUNT "log_recovery_redo_worker_count"
```

Comment:
Maybe a shorter name?
```suggestion
#define PRM_NAME_LOG_RECOVERY_REDO_WORKER_COUNT "recovery_parallel_count"
```

Reply:
Done